Content.Server/Chat/SuicideSystem.cs

```diff
@@ -123,19 +131,42 @@ private bool EnvironmentSuicideHandler(EntityUid victim, SuicideEvent suicideEve
             return false;
         }
 
-        private void ApplyDeath(EntityUid target, SuicideKind kind)
+        private void ApplyDeath(EntityUid target, string? kind, DamageSpecifier? damage)
```

Comment:
I would rename this function "ApplyLethalDamage" since that's what the function does

Reply:
instead of `string? kind` just use `ProtoId<DamageTypePrototype> kind = "Blunt"` and get rid of the declared fallback variable.

Reply:
Add a comment explaining that the function can work by either giving a single DamageType in the form of kind or varied damage types in the form of a DamageSpecifier. Declaring "kind" also overrides any passed along DamageSpecifier.

Reply:
Or just split it into two functions with a different overload.
`ApplyLethalDamage(EntityUid target, ProtoId<DamageTypePrototype> kind)` and `ApplyLethalDamage(EntityUid target, DamageSpecifer damage)` rather than trying to force it altogether.